### PR TITLE
[close #175] Add br apiv2 integration test

### LINKED ISSUE #175: br: Add integration test case to verify correctness of `backup-ts`
## Feature Request

**Is your feature request related to a problem? Please describe:**

**Describe the feature you'd like:**

**Describe alternatives you've considered:**

Add integration test case to verify correctness of `backup-ts`.
Refer to https://github.com/tikv/migration/pull/156.

**Teachability, Documentation, Adoption, Migration Strategy:**

### PR DESCRIPTION
**Pull request by haojinming**
### What problem does this PR solve?

Issue Number: close #175 

Problem Description: 

### What is changed and how does it work?
1. Add apiv2 integration test
2. Check backup ts.

### Code changes

- Has exported function/method change

### Check List for Tests

This PR has been tested by at least one of the following methods:
- Integration test
- Manual test (add detailed scripts or steps below)

### Side effects

- No side effects

### Related changes

- No related changes

### CODE REVIEW

File: .github/workflows/ci-br.yml [hunk 1]
```diff
@@ -74,12 +75,15 @@ jobs:
...
           /home/runner/.tiup/bin/tiup playground ${{ matrix.tikv_version }} --mode tikv-slim --kv 1 --without-monitor --kv.config /home/runner/work/migration/migration/.github/config/br_rawkv.toml --pd.config /home/runner/work/migration/migration/.github/config/br_pd.toml &> raw.out 2>&1 &
           # The first run of `tiup` has to download all components so it'll take longer.
           sleep 1m 30s
           # Parse PD address from `tiup` output
           echo "PD_ADDR=$(cat raw.out | grep -oP '(?<=PD client endpoints: \[)[0-9\.:]+(?=\])')" >> $GITHUB_ENV
+          # Output the api version
+          echo "API_VERSION=${{ matrix.api_version }}" >> $GITHUB_ENV
           # Log the output
           echo "$(cat raw.out)" >&2
       - name: run integration test
```
Comment (pingyu) [inline .github/workflows/ci-br.yml:89]: Suggest to add a step to print BR log after this. Whether failpoint is run as expected can only be verified by BR log now.

File: br/tests/rawkv/main.go [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+
+var (
+	maxMsgSize   = int(128 * units.MiB) // pd.ScanRegion may return a large response
+	maxBatchSize = uint(1024)           // max batch size with BatchPut
+
+	keyCnt        = flag.Uint("keycnt", 10000, "KeyCnt of testing")
+	pdAddr        = flag.String("pd", "127.0.0.1:2379", "Address of PD")
+	apiVersionInt = flag.Uint("api-version", 1, "Api version of tikv-server")
+	br            = flag.String("br", "br", "The br binary to be tested.")
+	brStorage     = flag.String("br-storage", "local:///tmp/backup_restore_test", "The url to store SST files of backup/resotre. Default: 'local:///tmp/backup_restore_test'")
```
[the remark this answers is not in the document]
Comment (haojinming) [reply br/tests/rawkv/main.go]: removed.

File: br/tests/rawkv/main.go [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+		return errors.Annotatef(err, "Backup fail:%s", backupOutput)
+	}
+	log.Info("backup finish:", zap.ByteString("output", backupOutput))
+	CheckBackupTS(apiVersion, tso, backupOutput, safeInterval)
+
+	err = tester.CleanData(ctx, prefix)
+	if err != nil {
+		return errors.Annotate(err, "Clean data fail")
+	}
+	restoreOutput, err := tester.Restore(ctx)
```
[the remark this answers is not in the document]
Comment (haojinming) [reply br/tests/rawkv/main.go:277]: added `scan` to verify the clean result.

File: .github/workflows/ci-br.yml (hunk 1 above)
Comment (haojinming) [reply .github/workflows/ci-br.yml:89]: Added.

**Review by pingyu — APPROVED** [review]
LGTM~

**Review by zeminzhou — APPROVED** [review]

**Pull request merged**